### DISCUSSION TITLE
Timing configuration options?

### DISCUSSION THREAD

**Discussion by b4shful** [Th3Un1q3/flipp_pomodoro#253] [Ideas]
Hi, great work on this app btw! I am finding more and more that I want my productivity related tools to be on a small device such as FZ without so many bells and whistles that it becomes distracting (like a pomodoro app on my phone might be, or a tab in my browser), so honestly love having the functionality on here!

I would love to see some configuration options kinda like https://pomofocus.io has, specifically with the timings:

![image](https://github.com/Th3Un1q3/flipp_pomodoro/assets/28313200/4a04fcdb-75ef-48c8-bba3-ab8a15967f8d)

Sometimes people find slightly adjusted timings to be personally more productive.

Pomofocus also allows pause, and while I understand to some people this may be against the spirit of the Pomodoro technique, if people find that it works better for them to be able to pause in the occasions when they have no other choice but to do so, it would be good as an option, maybe a setting for "allow pause" could be added so that people have the choice whether to have a pause button or not?

I would love to contribute instead of just suggesting such an idea, but unfortunately I am in the middle of college exams and am already procrastinating enough 😆 

Thanks and much love ❤️

**Comment by Th3Un1q3** [comment]
Hello @b4shful 

Appreciate your feedback on the app! Glad it's helping your productivity.

Your suggestion for customizable timings and a pause feature is noted. Balancing the Pomodoro technique's principles with user flexibility sounds like a good plan. Your insights are valued by me as the only contributor to the app right now.

Focus on your exams for now, and best of luck! Feel free to connect later if you're still interested in contributing.

**Comment by Th3Un1q3** [comment]
After years of waiting, @thevan4 finally added the feature #372 👏

Let's celebrate this contribution by testing it in the latest release and sending your kudos to Ivan. Also, fix any bugs you find as part of the celebration.

Download here:
 https://github.com/Th3Un1q3/flipp_pomodoro/releases/tag/v2.3.0
